src/global/mixins.scss

```diff
@@ -55,3 +55,39 @@
     height: 0;
   }
 }
+
+@mixin scrollbar($color: 'light') {
```

Comment:
Since there are only really two cases, I would prefer to have maybe two mixins `scrollbar-light` and `scrollbar-dark`, but I am also open to other suggestions.
(I would like to avoid the need to check every time, what parameter should be used to apply the other case.)

Reply:
I tried to find an alternative to this problem. I introduced the `scrollbar-ligh`t and `scrollbar-dark` mixins, but you can also use the simple `scrollbar` mixin which will take the currentcolor by default, otherwise it will take the color defined through the variables `-sbb-scrollbar-color` and `--sbb-scrollbar-color-hover`, where you can pass any color to customize the scrollbar before including the mixin in your CSS.

It would be helpful if you could use currentcolor with opacity to automatically define the scrollbar color or at least manage opacity in a simpler way. But the `opacity` property doesn't work in Chrome (strangely, it only works in Safari).